strawberryfields/io/xir_io.py

```diff
@@ -87,12 +117,9 @@ def from_xir(xir_prog):
                     params = sfpar.par_convert(params, prog)
                     gate(*params) | regrefs  # pylint:disable=expression-not-assigned
             else:
-                if callable(gate):
-                    gate() | regrefs  # pylint:disable=expression-not-assigned,pointless-statement
-                else:
-                    gate | regrefs  # pylint:disable=expression-not-assigned,pointless-statement
+                gate() | regrefs  # pylint:disable=expression-not-assigned,pointless-statement
```

Comment:
These Pylint comments make you question the original `|` [DSL](https://en.wikipedia.org/wiki/Domain-specific_language) a bit.

Reply:
I agree. I dislike needing them, but I guess that's to be expected when "hacking" syntax changes into another language.